test_util/integration_test.py

```diff
@@ -1290,75 +1292,105 @@ def wait_for_endpoint():
 
     r = requests.get(test_cache_url)
 
+    # Handy for diagnosing strange signal service test behavior, not sure if we should
+    # leave these in for master branch, or remove for the final PR.
+    print('TESTING SIGNAL RETURN:\n{}'.format(r.text))
+    print('CACHE SERVER STATUS:\n{}'.format(r.status_code))
+
     r_data = json.loads(r.json())
 
     cluster.destroy_marathon_app(signal_app_definition['id'])
     cluster.destroy_marathon_app(test_server_app_definition['id'])
 
     exp_data = {
-            'Event': 'health',
-            'UserId': '',
-            'ClusterId': '',
-            'Properties': {
-                'provider': 'onprem',
-                'source': 'cluster',
-                'clusterId': '',
-                'customerKey': '',
-                'environmentVersion': '',
-                'variant': 'open'}
-            }
+        'diagnostics': {
+            'event': 'health',
+            'anonymousId': 'open /var/lib/dcos/cluster-id: no such file or directory',
```

Comment:
Best is probably, since we're triggering the signal service manually, pass the option to change the location of the cluster-id file to one of our choosing, so we can verify the file gets loaded up, data sent properly.